Automattic/wp-calypso
# Checkout: Try to parse some known postal codes

Users shouldn't know all of the postal code specifics. As long as
they enter the right length, we should be able to add the space/dash
or whatever else is required.

Test:
- Go to Domains
- Pick a domain registration
- Edit Contact Info
- Contry = United Kingdom
- Type a UK postal code `wc1r4pf`
- Make sure the onBlur fires after selecting another field
- Make sure the postal code is formatted properly

Fixes https://github.com/Automattic/wp-calypso/issues/8681
Fixes https://github.com/Automattic/wp-calypso/issues/23752
